consensus/ibft/ibft.go

```diff
@@ -1290,7 +1246,14 @@ func (i *Ibft) verifyHeaderImpl(snap *Snapshot, parent, header *types.Header) er
 			return err
 		}
 
-		if err := verifyParentCommittedSeal(parentSnap, parent, header, i.quorumSize(header.Number)); err != nil {
+		err = i.signer.VerifyParentCommittedSeal(
+			parentSnap.Set,
+			parent,
+			header,
+			i.quorumSize(header.Number),
+		)
+
+		if err != nil {
 			return fmt.Errorf("failed to verify ParentCommittedSeal: %w", err)
 		}
```

Comment:
You can inline this

Reply:
Fixed in c04ef82713935e921b1f320d7788a70dde7cf307